bundles/org.openhab.binding.panamaxfurman/src/main/java/org/openhab/binding/panamaxfurman/internal/protocol/ProtocolMapper.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+/**
+* Exception thrown when an unsupported command type is sent to a channel.
+*
+* @author Dave Badia- Initial contribution
+*
+*/
+package org.openhab.binding.panamaxfurman.internal.protocol;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.panamaxfurman.internal.protocol.event.PanamaxFurmanInformationReceivedEvent;
+import org.openhab.core.types.State;
+
+/**
+ * Exception thrown when an unsupported command type is sent to a channel.
+ *
+ * @author Dave Badia - Initial contribution
+ */
+@NonNullByDefault
+public interface ProtocolMapper {
```

Comment:
Please adjust the javadoc in this interface for all methods, please list all parameters (return is allready done)